test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js

```diff
@@ -65,10 +77,9 @@ describe('v2/view-builder/views/webauthn/ChallengeWebauthnView', function () {
     expect(this.view.$('.idx-webauthn-verify-text').text()).toBe(
       'You will be prompted to use a security key or biometric verification (Windows Hello, Touch ID, etc.). Follow the instructions to complete verification.'
     );
-    expect(this.view.$('.retry-webauthn').css('display')).toBe('none');
-    expect(this.view.$('.retry-webauthn').text()).toBe('Retry');
+    expect(this.view.$('.retry-webauthn').css('display')).toBe('inline');
+    expect(this.view.$('.retry-webauthn').text()).toBe('Verify');
     expect(this.view.$('.webauthn-not-supported').length).toBe(0);
-    expect(this.view.$('.okta-waiting-spinner').css('display')).toBe('block');
```

Comment:
Removed the spinner check since it is hidden in css and css is not loaded in unit test. In e2e webauthn api is not loaded so spinner is not even added. Wil add a test in okta-core to check for spinner.